From 7cf1e7b7e9de7112a5211e2fe89e019f715f6d3e Mon Sep 17 00:00:00 2001
From: Bo Shen <voice.shen@atmel.com>
Date: Fri, 6 Mar 2015 13:10:30 +0800
Subject: [PATCH] Board: at91sam9g10ek: add allow 100MHz MCK

When at91sam9g10 work at 200MHz PCK, the MCK should be 100MHz.
So, allow the 100MHz MCK.

Signed-off-by: Bo Shen <voice.shen@atmel.com>
Acked-by: Nicolas Ferre <nicolas.ferre@atmel.com>
---
 board/at91sam9g10ek/Config.in.board | 1 +
 1 file changed, 1 insertion(+)

diff --git a/board/at91sam9g10ek/Config.in.board b/board/at91sam9g10ek/Config.in.board
index d146256..4809326 100644
--- a/board/at91sam9g10ek/Config.in.board
+++ b/board/at91sam9g10ek/Config.in.board
@@ -13,5 +13,6 @@ config	CONFIG_AT91SAM9G10EK
 	select ALLOW_DATAFLASH_RECOVERY
 	select ALLOW_NANDFLASH_RECOVERY
 	select SUPPORT_BUS_SPEED_133MHZ
+	select SUPPORT_BUS_SPEED_100MHZ
 	help
 	    Use the AT91SAM9G10EK Development board
-- 
1.9.1

